swagger.yaml

```diff
@@ -657,7 +657,7 @@ paths:
     get:
       operationId: findDeviceSurvey
       summary: Find a Device Survey.
-      description: Finds a Device Survey by id. Devices access this endpoint.
+      description: Finds a Device Survey by id.
       tags:
         - DeviceSurveys
       security:
```

Comment:
Remove this endpoint specific authentication definition now that devices don't access this endpoint anymore.

Reply:
Done